dat/scripts/pilotname/human.lua

```diff
@@ -0,0 +1,951 @@
+local fmt = require "format"
+
+-- For translation, just transliterate if necessary.
+local articles = {
```

Comment:
Seems to have lots of unused code (likely from copy pasting at the beginning). Should be made minimum.

Reply:
I also pushed to the `escorts` branch on this repo some minor fixes. You might want to pull them. They are basically warnings from luacheck, although there's still a lot left.